nms/app/views/equipment/FEGGatewayTable.tsx

```diff
@@ -65,16 +80,36 @@ export default function GatewayTable() {
             justifyContent="flex-end"
             alignItems="center"
             spacing={1}>
+            {currentView !== UPGRADE_VIEW && (
+              <Grid item>
+                <AutorefreshCheckbox
+                  autorefreshEnabled={refresh}
+                  onToggle={() => setRefresh(current => !current)}
+                />
+              </Grid>
+            )}
+            <Grid item>
+              <Text variant="body3">View</Text>
+            </Grid>
             <Grid item>
-              <AutorefreshCheckbox
-                autorefreshEnabled={refresh}
-                onToggle={() => setRefresh(current => !current)}
+              <TypedSelect
+                input={<OutlinedInput />}
+                value={currentView}
+                items={{
+                  STATUS: 'Status',
+                  UPGRADE: 'Upgrade',
+                }}
+                onChange={setCurrentView}
               />
             </Grid>
           </Grid>
         )}
       />
-      <StatusTable refresh={refresh} />
+      {currentView === 'UPGRADE' ? (
```

Comment:
Compare to [line 83](https://github.com/magma/magma/pull/13456/files#diff-d8679aafc0dd22fd8dffd050f2e322b40dc01b2349acc81453b93b606de243eaR83). This seems to be inconsistent.